packages/lodestar-params/src/utils.ts

```diff
@@ -1,29 +1,90 @@
 import {FAILSAFE_SCHEMA, Schema, Type} from "js-yaml";
-import {TypeMap, typeMap} from "./types";
 import {IBeaconParams} from "./interface";
-import * as constants from "./constants";
+import {ContainerType, BigIntUintType, NumberUintType, BitVectorType, BitListType, Json} from "@chainsafe/ssz";
 
-export function createIBeaconParams(params: Record<string, unknown>): Partial<IBeaconParams> {
-  return convertTypes(params, typeMap);
-}
+const beaconParamsType = new ContainerType({
+  fields: {
+    // Misc
+    MAX_COMMITTEES_PER_SLOT: new NumberUintType({byteLength: 8}),
```

Comment:
You should factor each unique type out above the `beaconParamsType` declaration:
```typescript
const Number64 = new NumberUintType({byteLength: 8});
...

const beaconParamsType = new ContainerType({
  fields: {
    MAX_COMMITTEES_PER_SLOT: Number64,
    ...
```